<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotosDD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="FileBox">
      <property name="title">
       <string>File Box</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="AddImageButton">
         <property name="toolTip">
          <string>Add one or more image files</string>
         </property>
         <property name="text">
          <string>Add Image(s)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>111</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="AddDirButton">
         <property name="toolTip">
          <string>Add all image files within a directory</string>
         </property>
         <property name="text">
          <string>Add Directory</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="ShowIconCheckBox">
         <property name="text">
          <string>Show Preview of chosen Images</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="ClearImageListButton">
         <property name="toolTip">
          <string>remove all images from list</string>
         </property>
         <property name="text">
          <string>Clear List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="RemoveImageButton">
         <property name="toolTip">
          <string>remove selected image from list</string>
         </property>
         <property name="text">
          <string>remove Image</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QListWidget" name="FilesList"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="taskProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="overallProgressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedKingdom"/>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTips"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_EasyImageSizer_GUI"/>
    <addaction name="actionAbout_EasyImageSizer"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMore_functions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="text">
    <string>Config ...</string>
   </property>
  </action>
  <action name="actionAbout_EasyImageSizer_GUI">
   <property name="text">
    <string>About EasyImageSizer-GUI</string>
   </property>
  </action>
  <action name="actionAbout_EasyImageSizer">
   <property name="text">
    <string>About EasyImageSizer</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTips">
   <property name="text">
    <string>Tips</string>
   </property>
  </action>
  <action name="actionMore_functions">
   <property name="text">
    <string>more functions ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>AddImageButton</tabstop>
  <tabstop>ShowIconCheckBox</tabstop>
  <tabstop>AddDirButton</tabstop>
  <tabstop>ClearImageListButton</tabstop>
  <tabstop>RemoveImageButton</tabstop>
  <tabstop>FilesList</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>commandLinkButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
